src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -2235,7 +2235,7 @@ void GameObject::BuildValuesUpdate(uint8 updateType, ByteBuffer* data, Player* t
 
             if (index == GAMEOBJECT_DYNAMIC)
             {
-                uint16 dynFlags = 0;
+                uint16 dynFlags = m_uint32Values[GAMEOBJECT_DYNAMIC];
```

Comment:
Explain this change please, it does not look like something that should be in a script PR.